app/config/parameters.yml.dist

```diff
@@ -41,6 +41,12 @@ parameters:
     paypal.express_checkout.signature: EDITME
     paypal.express_checkout.sandbox: true
 
+    paypal.pro_checkout.username: EDITME
+    paypal.pro_checkout.password: EDITME
+    paypal.pro_checkout.partner: EDITME
+    paypal.pro_checkout.vendor: EDITME
+    paypal.pro_checkout.sandbox: true
```

Comment:
@pjedrzejewski  :+1:  to make it configurable via backend, would be a great step further